Create ui-c8y-1019-23-1-tfa-strategy-updated-by-tfa-endpoint-6146-graft-release-cd-6211.md (#1994)
SoftwareAG/c8y-docs: content/change-logs/platform-services/ui-c8y-1019-23-1-tfa-strategy-updated-by-tfa-endpoint-6146-graft-release-cd-6211.md

---
date: ""

> title: The update of the TFA strategy takes place through the TFA endpoint.

Reviewer: ```suggestion
title: Changing the TFA method for a tenant now updates it for all users
```

> title: tfa strategy updated by tfa endpoint (#6146) [GRAFT][release/cd] (#6211)

Author: The Claude AI suggested to use the following title:
```suggestion
title: Update TFA strategy via TFA endpoint
```

product_area: Platform services
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:

>   - value: component-0UgqXH1Ys

Reviewer: ```suggestion
  - value: q3kclF6pO 
```

>     label: Administration

Reviewer: ```suggestion
    label: Authentication
```

build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-52862
version: 1019.23.1
---

> tfa strategy updated by tfa endpoint (#6146) [GRAFT][release/cd] (#6211)

Author: The Claude AI suggested to use the following body:
```suggestion
In an effort to enhance the security and flexibility of the two-factor authentication (TFA) process, the TFA strategy can now be updated using the dedicated TFA endpoint. This change allows users to modify their TFA settings more easily and efficiently, without needing to navigate through multiple steps or menus. The updated functionality streamlines the TFA configuration process, providing a more user-friendly experience while maintaining the robust security measures that TFA offers.
```

> When changing the TFA strategy, the application must update the strategy for all users. This effect is not implemented if the strategy update occurs through changing tenant options. Changing the endpoint ensures that after changing the strategy on the configuration page, the strategy for all users will be updated.

Reviewer: ```suggestion
When changing the TFA method for a tenant, the application must update the method for all users. This did not happen if the method was changed through the tenant options. Changing the method through the TFA endpoint now ensures that after changing the method on the configuration page, the method will be updated for all users.
```

Reviewer 2: @BeateRixen I like your suggested change much more
